### PR TITLE
Refactor app sections

### PR DESCRIPTION
**Pull request by gsanchietti**
- Promote existing apps
- Move old doc to legacy section
- Reorder apps index section

Preview: https://nethserver.docs.nethesis.it/en/refactor_apps/index.html

Replaces #553

### CODE REVIEW

File: administrator-manual/en/freepbx.rst [hunk 1]
```diff
@@ -2,29 +2,35 @@
...
-
 :index:`FreePBX` is a web-based open source GUI (graphical user interface)
 that controls and manages :index:`Asterisk` (PBX), an open source communication server
 (https://www.freepbx.org/).
 
 Installation
 ============
 
-You can install FreePBX from the package manager of NethServer, the module
+You can install FreePBX from the package manager of |product|, the module
```
Comment (DavidePrincipi) [inline administrator-manual/en/freepbx.rst:12]: For nsent FreePBX is bundled with NethVoice and must not be installed separately

File: administrator-manual/en/team_chat.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ If your server is using a Let's Encrypt certificate as default, make also sure t
 
 How to configure:
 
-1. Access :guilabel:`Team chat` page inside the Server Manager
+1. Click on :guilabel:`Settings` buton of **Mattermost** application inside the *Applications* page
```
Comment (DavidePrincipi) [inline administrator-manual/en/team_chat.rst:26]: Typo

```suggestion
1. Click on :guilabel:`Settings` button of **Mattermost** application inside the *Applications* page
```

File: administrator-manual/en/freepbx.rst (hunk 1 above)
Comment (gsanchietti) [reply administrator-manual/en/freepbx.rst:12]: FreePBX is not visible inside the Enterprise Software Manager.
By the way, if you install it before NethVoice, nothing will break.

File: administrator-manual/en/nextcloud.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ Installation
 The installation can be done through the |product| web interface.
 After the installation:
 
-* open the URL https://your_nethserver_ip/nextcloud
+* click on :menuselection:`Nextcloud` application title inside the :menuselection:`Applications` page or open the URL https://your_nethserver_ip/nextcloud
```
Comment (DavidePrincipi) [inline administrator-manual/en/nextcloud.rst:26]:
```suggestion
* click on the :menuselection:`Applications > Nextcloud` item title or open the URL ``https://your_nethserver_ip/nextcloud``
```

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**